docs/user-guide/system-admin/configuration/mac-installation.md

```diff
@@ -0,0 +1,52 @@
+## What you need
+
+- Make sure JDK is installed, [download it here](www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html).
+- JBoss Enterprise Application Platform 6.4 (EAP). This is the recommended container for Zanata, and it can be [downloaded here](http://www.jboss.org/jbossas/downloads/). [Installation instructions](http://www.jboss.org/products/eap/get-started/#!project=eap).
+- Wildfly is much easier though (recommended version is 8.1.x) which can be [downloaded here](http://wildfly.org/downloads/). You can also install with `brew install wildfly-as`.
```

Comment:
Line 4 and 5 are the same thing. Suggest changing structure to:

```
- Supported JBoss or Wildfly server.
   - JBoss Enterprise Application Platform 6.4 (EAP). This is the recommended container for Zanata, and it can be downloaded from [jboss.org downloads](http://www.jboss.org/jbossas/downloads/). [Installation instructions](http://www.jboss.org/products/eap/get-started/#!project=eap).
   - Wildfly is much easier to install on Mac. Recommended version is 8.1.x, which can be downloaded from [wildfly.org downloads](http://wildfly.org/downloads/). You can also install with `brew install wildfly-as`.
```